honeybee_vtk/model.py

```diff
@@ -299,15 +292,41 @@ def to_vtkjs(self, folder: str = '.', name: str = None) -> str:
         # add sensor grids
         # it is separate from other DATA_SETS mainly for data visualization
         data = self.sensor_grids
-        path = data.to_folder(temp_folder)
+        path = data.to_folder(point_in_time_folder)
         if path:
             scene.append(data.as_data_set())
 
         # write index.json
         index_json = IndexJSON()
         index_json.scene = scene
-        index_json.to_json(temp_folder)
+        index_json.to_json(point_in_time_folder)
+
+        return point_in_time_folder
+
+    @staticmethod
+    def to_vtkjs(temp_folder: pathlib.Path, folder: str = '.', name: str = None) -> str:
```

Comment:
This is not a good move in my opinion from the API point of view. `to_vtkjs` is what we have used as the standard method to write the model to a vtkjs file. Making it a static second-level method is not what I would do.

In any case, it is strange to have temp_folder as a required input. Moreover we should either use strings or pathlib.Path for folder inputs. It looks so random that the first one is expecting a Path and the second folder is a string.